### PR TITLE
Mapped a bunch of random stuff

### PR DESCRIPTION
**Pull request by Darkhax**
Mapped a little bit of everything.

### CODE REVIEW

File: mappings/none/bzq.mapping [hunk 1]
```diff
@@ -1,5 +1,9 @@
 CLASS none/bzq
+	METHOD a useAmbientOcclusion ()Z
 	METHOD a getQuads (Lnone/asj;Lnone/cu;J)Ljava/util/List;
 		ARG 0 state
 		ARG 1 face
 		ARG 2 rand
+	METHOD b isThreeDimensionalInGUI ()Z
```
Comment (asiekierka) [inline mappings/none/bzq.mapping:7]: hasDepthInGui? isThreeDimensionalInGui is a bad name.

File: mappings/none/bzq.mapping [hunk 2]
```diff
@@ -1,5 +1,9 @@
 CLASS none/bzq
+	METHOD a useAmbientOcclusion ()Z
```
Comment (asiekierka) [inline mappings/none/bzq.mapping:2]: usesAmbientOcclusion? isAmbientOcclusionEnabled?

File: mappings/none/bzq.mapping [hunk 3]
```diff
@@ -1,5 +1,9 @@
...
 	METHOD a getQuads (Lnone/asj;Lnone/cu;J)Ljava/util/List;
 		ARG 0 state
 		ARG 1 face
 		ARG 2 rand
+	METHOD b isThreeDimensionalInGUI ()Z
+	METHOD c isBuiltin ()Z
```
Comment (asiekierka) [inline mappings/none/bzq.mapping:8]: isBuiltinModel

File: mappings/net/minecraft/reference/Sounds.mapping [hunk 4]
```diff
@@ -7,24 +7,24 @@ CLASS none/nl net/minecraft/reference/Sounds
 	FIELD F ENTITY_BLAZE_SHOOT Lnone/nk;
 	FIELD G ENTITY_BOBBER_SPLASH Lnone/nk;
 	FIELD H ENTITY_BOBBER_THROW Lnone/nk;
-	FIELD I ITEM_BOTTLE_FILL Lnone/nk;
```
Comment (asiekierka) [inline mappings/net/minecraft/reference/Sounds.mapping:10]: Just grab the old Sounds.mapping (revert) and don't touch this manually. Text editors are better for these kinds of mass movements.

**Comment by asiekierka** [comment]
Amazing work, but we broke it I'm afraid!

**Comment by asiekierka** [comment]
I'll fix it up myself.

File: mappings/none/bzq.mapping (hunk 1 above)
Comment (shadowfacts) [reply mappings/none/bzq.mapping:7]: Just `hasDepth` IMO

File: mappings/none/bzq.mapping (hunk 2 above)
Comment (shadowfacts) [reply mappings/none/bzq.mapping:2]: `usesAmbientOcclusion` IMO so then the call is something like `if (gui.usesAmbientOcclusion())`

**Pull request closed without merging**